docs/content/tutorial/advanced-tutorial/types.mdx

```diff
@@ -22,8 +22,8 @@ Let's look back at our simple `read_csv` solid.
 
 ```python file=/intro_tutorial/basics/e04_quality/inputs_typed.py startafter=start_inputs_typed_marker_0 endbefore=end_inputs_typed_marker_0
 @solid
-def read_csv(context, csv_path: str):
-    csv_path = os.path.join(os.path.dirname(__file__), csv_path)
```

Comment:
Here's a case where the tutorial section doesn't seem to work "as is" without adding in some of the elided content from the docs snippets. 
I'm going to revert this from this PR (not counting the pathlib use).